tracee/tracee.go

```diff
@@ -765,7 +765,7 @@ func (t *Tracee) initBPF(bpfObjectPath string) error {
 
 	supportRawTracepoints, err := supportRawTP()
 	if err != nil {
-		return fmt.Errorf("Failed to find kernel version: %v", err)
```

Comment:
The errors that I write are usually lowercased, as mentioned here: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

we do have places where errors are mistakenly styles like a sentence, if you want you could change those instead

Reply:
Hmm I didn't understand your comment, are you saying this is more of a log statement than an error itself? So in terms of code is your suggestion:

```
log.Printf("Failed to find kernel version: %v", err)
return err
```

Reply:
again, read this the wrong way :)